src/test/ui/pub/issue-33174-restricted-type-in-public-interface.rs

```diff
@@ -16,11 +16,11 @@ pub struct Shell<T> {
 }
 
 pub type Helix_pomatia = Shell<Snail>;
-//~^ ERROR crate-visible type `Snail` in public interface
-//~| NOTE can't leak crate-visible type
+//~^ ERROR private type `Snail` in public interface
+//~| NOTE can't leak private type
 pub type Dermochelys_coriacea = Shell<sea::Turtle>;
-//~^ ERROR restricted type `Turtle` in public interface
-//~| NOTE can't leak restricted type
+//~^ ERROR crate-private type `Turtle` in public interface
+//~| NOTE can't leak crate-private type
```

Comment:
Are these changes intentional?

Reply:
The wording is now based on semantic visibilities returned by `tcx.visibility()`.
Previously it was based on syntactic visibilities.
You can see that the visibility in question is actually crate-private (`pub(crate)`), even if it's written differently syntactically.